tests/operators/test_python.py

```diff
@@ -375,6 +375,20 @@ def func():
             "INFO:airflow.task.operators:Done. Returned value not shown" in cm.output
         ), "Log message that the option is turned off should be shown"
 
+    def test_python_operator_templates_exts(self):
+        def func():
+            return "test_return_value"
+
+        python_operator = PythonOperator(
+            task_id="python_operator",
+            python_callable=func,
+            dag=self.dag,
+            show_return_value_in_logs=False,
+            templates_exts=['test_ext']
+        )
+
+        assert python_operator.template_ext == ['test_ext']
```

Comment:
Why is this test relevant? `tempalte_ext` is a property on BaseOperator, is it not?

Reply:
Oh it’s the `templates_exts` argument specific to PythonOperaotr. I wonder if it’s a good target for deprecation…